specification/conformance/conformance_rules/columns/chargefrequency.json

```
@@ -0,0 +1,148 @@
+{
+  "ChargeFrequency-C-000-O": {
+    "Function": "Composite",
+    "Reference": "ChargeFrequency",
+    "EntityType": "Column",
+    "Notes": "Root composite – most restrictive child is Conditional (C)",
```

I think we can kill this note. 
```suggestion
    "Notes": "",
```